### ISSUE TITLE
Use recaptcha on user signup

### ISSUE THREAD

**Issue by RyanTG** [scottwainstock/pbm#1195]
~~https://github.com/ambethia/recaptcha~~

**Comment by RyanTG** [comment]
lil hard to test this on local. https://github.com/scottwainstock/pbm/pull/1196/commits/1df80032392f1980fb48f49b66e6e11c00c36dcb

But I think that'll be good? I didn't add any `if verify_recaptcha` stuff, a la https://github.com/scottwainstock/pbm/blob/11eb2445f2a712d195ca0b0fb61a43940aa6a249/app/controllers/pages_controller.rb#L106

But perhaps I need to...

**Comment by RyanTG** [comment]
To catch up:

We are already using recaptcha, so I didn't need to link to it in the original post. I attempted to set it up. But it didn't work because I added it to a def that's in the api, which the website registration doesn't even use (but the app uses). So all I did was break the registration on the app.

Here's my best guess at a solution: a new def needs to be added to https://github.com/scottwainstock/pbm/blob/12c81110b3549893b5d3a4e3f756f7a6df81fcc7/app/models/user.rb for registration, that includes `if verify_recaptcha`
